src/blr/rules/trim.smk

```diff
@@ -1,48 +1,70 @@
 """
 Rules connected to trimming of FASTQ files.
+
+READ1 LAYOUT
+
+5'-CAGTTGATCATCAGCAGGTAATCTGGBDVHBDVHBDVHBDVHBDVHCATGACCTCTTGGAACTGTCAGATGTGTATAAGAGACAGNNNN...NNNN(CTGTCTCTTATACACATCT)-3'
+   <------------h1----------><-------DBS--------><-----------------h2------------------><---gDNA--><---------h3-------->
+    h1 may inlude frameshift                                                                        Presence depencd on
+    oligos varying from 0-4                                                                         insert length
+    extra oligos in 5' end
+
+READ 2 LAYOUT
+
+5'-NNNN...NNNN(CTGTCTCTTATACACATCT)-3'
+   <---gDNA--><---------h3-------->
+              Presence depencd on
+              insert length
 """
 
+h1="CAGTTGATCATCAGCAGGTAATCTGG"
```

Comment:
Please add spaces around operators

Reply:
This is a really helpful ASCII drawing. Can you please also write what h1 to h3 and DBS mean? (I can guess, but I shouldn’t have to.) This could perhaps also be moved to the documentation at some point.

Reply:
Since I moved these to the config file I added a small comment about them. Hopefully they should be easier to understand. I agree that the drawing could be useful in the docs, I included it here just to keep my own sanity while doing changes :)